apps/daemon/src/websocket.ts

```
@@ -0,0 +1,68 @@
+let wss: WebSocketServer | null = null;
+const messageHandlers = new Set<MessageHandler>();
+
+export function setupWebSocket(server: Server): WebSocketServer {
+  wss = new WebSocketServer({ server, path: '/ws' });
+
+  wss.on('connection', (ws) => {
+    console.log('[WebSocket] Client connected. Total:', wss!.clients.size);
+
+    ws.on('message', (raw) => {
+      const msg = JSON.parse(raw.toString()) as ClientMessage;
+      messageHandlers.forEach((handler) => handler(msg));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)